### DISCUSSION TITLE
Help me please / query raw with builder

### DISCUSSION THREAD

**Discussion by marquezm2410** [prisma/prisma#25134] [Q&A]
### Question

Hello, I hope you are well, Prisma supports something similar to these examples that I put below? I want to stop using native SQL views, which has been the only solution found so far, I have not found a way to combine complex queries using prisma.$queryRaw and the prisma query creator api, what I want to do is something similar To this, in the example I put a simple expression (id * 5) but in real life I have had to create more complex ones such as calling functions, obtaining values ​​from other tables, etc.

prisma.$queryRaw`SELECT id, name, (id * 5) as columnAdded FROM User`.builder({
        where :{id : 1},
        include: { posts : true},
        skip: ...,
        take: ...,
        orderBy :{ columnAdded : 'desc' }
      })

      Or

      prisma.users.findMany({
        select: {id: true, name: true, columnAdded: {expression: '(id * 5) as columnAdded '},  posts : true }
        where :{...},
        skip: ...,
        take: ...,
        orderBy :{ columnAdded : 'desc' }
      })

   Is it possible to achieve a result similar to these examples?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @marquezm2410 👋 

Prisma Client does not support embedding SQL expressions directly within its query API. However, you can achieve similar results using raw SQL queries with [$queryRaw](https://www.prisma.io/docs/orm/prisma-client/using-raw-sql/raw-queries). 

You can use `$queryRaw` to execute raw SQL queries. This allows you to include complex expressions and functions in your queries but you should know that these queries are not type safe. You can use the newer [TypedSQL](https://www.prisma.io/docs/orm/prisma-client/using-raw-sql) preview feature to write type-safe queries.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
